code/src/Core/Language.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Templates.Core
+{
+    public static class Language
```

Comment:
should this be "ProgrammingLanguages",  I think it should be a plural due to Colors.  https://docs.microsoft.com/en-us/uwp/api/windows.ui.colors